### PR TITLE
Create android-apk

### PR DESCRIPTION
**Pull request by bennshine**
creating apk

## checklist:

- [ ] New feature
- [ ] Bug fix
- [x] Tested locally

### CODE REVIEW

File: android-apk [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Android App Release Build
```
Comment (thomasread99) [inline android-apk]: Please could you change the name to `Android App APK Build`

File: android-apk [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+on:
+  push:
+    branches:
+      - main
```
Comment (thomasread99) [inline android-apk]:
```suggestion
on:
  # Specify when you want the build to run here
```

File: android-apk [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build Android app
+        run: eas build --platform android --profile preview --local --output ${{ github.workspace }}/app-release.apk
+
+      - name: Upload AAB artifact
```
Comment (thomasread99) [inline android-apk]: Please can you rename this step to `Upload APK artifact`?

**Review by thomasread99 — CHANGES_REQUESTED** [review]
Looks good, just a couple of minor naming things. Please could you also:

- move the file to the android folder, and call it android-apk-build.yml
- use conventional commit messages (see CONTRIBUTING.md for more information). To edit your existing commit, use `git commit --amend` and the text editor will pop up allowing you to edit the commit message

**Review by bennshine — COMMENTED** [review]
rename file
changed the suggestion

**Review by thomasread99 — APPROVED** [review]

**Pull request merged**